### PR TITLE
GF 7 and jakarta ee 10 support.

### PR DESCRIPTION
**Pull request by gurunrao**
GF 7 and jakarta ee 10 support.
Signed-off-by: gurunandan.rao@oracle.com <gurunandan.rao@oracle.com>

Full pass for GF 7 with BV TCK is expected post resolution of https://github.com/eclipse-ee4j/glassfish/issues/23872
CI run - https://ci.eclipse.org/jakartaee-tck/job/guru/job/bvtck-porting-guru/job/jakartaee10/9/

### CODE REVIEW

File: glassfish-tck-runner/pom.xml [hunk 1]
```diff
@@ -215,17 +228,32 @@
...
 	        <groupId>org.glassfish.jersey.inject</groupId>
 	        <artifactId>jersey-hk2</artifactId>
-	        <version>3.0.0</version>
+	        <version>3.0.3</version>
         </dependency>
         <dependency>
 	        <groupId>org.glassfish.hk2</groupId>
 	        <artifactId>hk2-locator</artifactId>
-	        <version>3.0.0</version>
+	        <version>3.0.3</version>
```
Comment (dmatej) [inline glassfish-tck-runner/pom.xml:236]: It would be perfect to use consistent formatting (not your mistake, previous version mixed tabs and spaces).

**Review by dmatej — COMMENTED** [review]
I am not sure about committing binary files, especially so large, do we have alternatives?

**Comment by gurunrao** [comment]
> I am not sure about committing binary files, especially so large, do we have alternatives?

we have followed same convention of using checked-in sig jar files, as in jakartaee-tck repository(and bvtck-porting kit previous version). We can enhance to download dependency in https://github.com/eclipse-ee4j/bvtck-porting/tree/master/lib as part of a new enhancement, since scope of PR is for GF7 and Jakarta EE 10 support.

File: glassfish-tck-runner/pom.xml (hunk 1 above)
Comment (gurunrao) [reply glassfish-tck-runner/pom.xml]: done

**Review by dmatej — APPROVED** [review]

**Comment by scottmarlow** [comment]
> > I am not sure about committing binary files, especially so large, do we have alternatives?
> 
> we have followed same convention of using checked-in sig jar files, as in jakartaee-tck repository(and bvtck-porting kit previous version). We can enhance to download dependency in https://github.com/eclipse-ee4j/bvtck-porting/tree/master/lib as part of a new enhancement, since scope of PR is of GF7 and Jakarta EE 10 support.

^ should work (as the same netbeans sigtest is used I think), although further pull requests would be welcome to introduce change to use sigtest jar included in Bean Validation TCK as mentioned via https://github.com/eclipse-ee4j/beanvalidation-tck/blob/master/documentation/src/main/asciidoc/sigtest.asciidoc, this could be a future change.

**Review by scottmarlow — APPROVED** [review]

**Pull request merged**